From 66842c96578102f140d72e464966d3cf0be12a2c Mon Sep 17 00:00:00 2001
From: Elaine Gibson <ypsvlq@gmail.com>
Date: Fri, 31 Jul 2026 13:08:51 +0100
Subject: [PATCH] Fix missing include

---
 filesystem/udf/udf_browser.ixx | 1 +
 1 file changed, 1 insertion(+)

diff --git a/filesystem/udf/udf_browser.ixx b/filesystem/udf/udf_browser.ixx
index 8f00860a..5a8d2a79 100644
--- a/filesystem/udf/udf_browser.ixx
+++ b/filesystem/udf/udf_browser.ixx
@@ -1,4 +1,5 @@
 module;
+#include <cstdint>
 #include <cstring>
 #include <memory>
 #include <optional>
